### DISCUSSION TITLE
Promotheus non operator installation as a project admin

### DISCUSSION THREAD

**Discussion by phaneesh11** [artifacthub/hub#4170] [Q&A]
**Describe the bug**

**To Reproduce**
I am following below steps on ocp4.16.where I have access to project and I am not a cluster admin, I have attached values.yaml
[Uploading prom-values.yaml.yaml.zip…]()

helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
helm install my-prometheus prometheus-community/prometheus --version 26.0.0

Getting below error
[root@ad cos]# helm install obs-prometheus prometheus-community/prometheus --version 26.0.0 --values values.yaml
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /root/.kube/config
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /root/.kube/config
Error: INSTALLATION FAILED: Unable to continue with install: could not get information about the resource ClusterRoleBinding "obs-prometheus-kube-state-metrics" in namespace "": clusterrolebindings.rbac.authorization.k8s.io "obs-prometheus-kube-state-metrics" is forbidden: User "rtmsadmin" cannot get resource "clusterrolebindings" in API group "rbac.authorization.k8s.io" at the cluster scope

**Expected behavior**
Installation at namespace level
**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by tegioz** [comment]
Hi @phaneesh11 👋

For issues with the content listed on [artifacthub.io](http://artifacthub.io/) it's best to contact the particular project directly. In the links section of the right panel in the package view you usually may find some links that can point you to the project's repository or the maintainers emails.

Hope this helps :slightly_smiling_face:
